levitate/remote-write/statsd/vmagent-graphite/README.md

```diff
@@ -1,2 +1,21 @@
 This folder contains demo code for statsd to Levitate remote write integration
 using vmagent.
+
```

Comment:
@saurabh-hirani Keeping this as-is. We don't have the corresponding section on https://docs.last9.io/docs/levitate-integrations-statsd for this.